### PR TITLE
Update R_Exercises8.Rmd

### PR DESCRIPTION
**Pull request by King0fKings**
Professor, I was able to fork the document, after I emailed you a few minutes ago.  But It appears I am not able to re-edit.

### CODE REVIEW

File: R_Exercises8.Rmd [hunk 1]
```diff
@@ -22,22 +22,65 @@ place. Do **NOT** rename the file.
...
+ *lentils
+ Cumin
+ Curry
 
 
 3. Insert an R code chunk below. Show the `summary` of the `Orange` dataset.
 Run a linear regression using the `lm` function to predict `circumference` from
 `age` and assign the result. Show the `summary` of the fit object that you
 assigned.
+orange_fit<-lm(circumference ~ age, data = Orange)
```
Comment (LiNk-NY) [inline R_Exercises8.Rmd:46]: Make sure you add a code chunk to indicate that you are providing R code

File: R_Exercises8.Rmd [hunk 2]
```diff
@@ -22,22 +22,65 @@ place. Do **NOT** rename the file.
...
 
 2. Add an unordered Markdown list of your favorite food dishes and
 add their ingredients as sub-items in the list. You should enter at least two
 dishes with two ingredients each.
+* Favorite Food. Taboli Salad
+  *quinoq
+  *parsley
+  olive oil
+  
+*Lentil Stew
```
Comment (LiNk-NY) [inline R_Exercises8.Rmd:36]: Work on the spacing of the list. See the examples in the lesson

File: R_Exercises8.Rmd [hunk 3]
```diff
@@ -22,22 +22,65 @@ place. Do **NOT** rename the file.
...
+
+
+
+
 
 
 4. Add an RMarkdown weblink (in link format) below to your favorite R reference
 website.
 
+https://rmarkdown.rstudio.com/
```
Comment (LiNk-NY) [inline R_Exercises8.Rmd:81]: This is not a markdown type of web link

**Comment by LiNk-NY** [comment]
I am guessing that you are Harriet @King0fKings 
You were to add your name in the author field in the document.

**Comment by King0fKings** [comment]
Good Morning Marcel,
Thank you.
It is Harriet. King0fKings.
I found a challenge in forking the test to the RStudio.  I kept on getting “Path” error.  I did the best I could.
Best regards.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Pull request closed without merging**